examples/ex14.cpp

```diff
@@ -19,6 +19,10 @@
 //               ex14 -m ../data/amr-hex.mesh
 //               ex14 -m ../data/fichera-amr.mesh
 //
+// Device sample runs:
+//               ex14 -pa -d cuda -o 3
+//               ex14 -pa -d cuda -o 3 -m ../data/fichera.mesh -o 3
```

Comment:
The arguments `-o 3` are repeated:
```suggestion
//               ex14 -pa -d cuda -o 3 -m ../data/fichera.mesh
```